src/com/engeto/CountryExtractor.java

```diff
@@ -0,0 +1,29 @@
+package com.engeto;
+
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONObject;
+
+import java.util.ArrayList;
+
+public class CountryExtractor {
+
+    public ArrayList<Country> extractCountries(JSONObject jsonObject) {
+        JSONArray list = jsonObject.getJSONArray("rates");
+        ArrayList<Country> countries = new ArrayList<>();
+
+        for (Object elements : list) {
+            JSONObject jsonElements = (JSONObject) elements;
+            String name = jsonElements.getString("name");
+            String code = jsonElements.getString("code");
+
+            JSONObject periods = jsonElements.getJSONArray("periods").getJSONObject(0);
```

Comment:
Ty tu predpokladas, ze prvy prvok v periods bude ten sucasny. Mozno je to tak, mozno nie, lepsie je to vsak osetrit v kodu (prejst cely array a vybrat periodu, ktora je aktualne v platnosti).